ProcessLib/InitialCondition.h

```diff
@@ -35,7 +35,7 @@ class InitialCondition
 {
 public:
 	virtual ~InitialCondition() = default;
-	virtual double getValue(MeshLib::Node const&) const = 0;
+	virtual double getValue(MeshLib::Node const&, int const) const = 0;
```

Comment:
Can you add variable names, so one knows, what the arguments mean? Also to the already existing arg.